lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -934,7 +934,9 @@ export class UserAgentApplication {
       let authErr: AuthError;
       let cacheResultResponse;
 
-      if (!userContainedClaims) {
+      const forceCacheSkip = request ? request.forceRefresh : false;
```

Comment:
This could be `!!request.forceRefresh` (as `request` should always be present, correct)?

Reply:
no the feedback was that it could be null

Reply:
I doubt that, as there is code above this that access properties on `request` with null checking, which would break if that was the case.

Reply:
So one case where ATS is called from login calls where request is optional. It is not specific to this case, we have to in general check for that path where request can be null. That was the intent of the original comment. If it is referenced before this statement, we need to check if that is a bug in itself.